.cloudcannon/routing.json

```
@@ -829,6 +829,140 @@
+      "status": 301
+    },
+    
+    {
+      "from": "/documentation/articles/introducing-your-site-navigation/",
+      "to": "/documentation/articles/configure-your-site-navigation/",
+      "status": 301
+    },
+    
+    {
+      "from": "/documentation/articles/creating-collection-schemas/",
+      "to": "/documentation/articles/create-a-schema/",
+      "status": 301
+    },
+
+    {
+      "from": "/documentation/articles/defining-the-default-contents-of-new-files/",
+      "to": "/documentation/articles/define-your-default-file-contents/",
+      "status": 301
+    },
+    
+    {
+      "from": "/documentation/articles/choosing-where-to-create-new-files/",
+      "to": "/documentation/articles/set-the-path-for-new-files/",
+      "status": 301
+    },
+    
+    {
+      "from": "/documentation/articles/defining-more-ways-to-add-new-files/",
+      "to": "/documentation/articles/configure-the-add-button-in-collections/",
+      "status": 301
+    },
+    
+    {
+      "from": "/documentation/articles/formatting-your-commit-messages/",
+      "to": "/documentation/articles/configure-your-commit-messages/",
+      "status": 301
+    },
+    
+    {
+      "from": "/documentation/articles/how-to-choose-what-input-is-used-in-the-data-editor/",
+      "to": "/documentation/articles/configure-your-inputs/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/defining-what-adds-to-an-array-with-array-structures/",
+      "to": "/documentation/articles/create-a-structure/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/using-a-modal-with-array-structures/",
+      "to": "/documentation/articles/create-a-structure/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/editing-with-docusaurus/",
+      "to": "/documentation/articles/snippets-using-docusaurus-components/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/editing-with-eleventy-shortcodes/",
+      "to": "/documentation/articles/snippets-using-eleventy-shortcodes/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/editing-with-hugo-shortcodes/",
+      "to": "/documentation/articles/snippets-using-hugo-shortcodes/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/editing-with-mdx-components/",
+      "to": "/documentation/articles/snippets-using-mdx-components/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/editing-with-python-markdown/",
+      "to": "/documentation/articles/snippets-using-python-markdown/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/defining-editable-regions-in-your-html/",
+      "to": "/documentation/articles/define-editable-regions-in-your-html/",
+      "status": 301
+    },
+    {
+      "from": "/documentation/articles/introducing-the-site-dashboard/",
+      "to": "/documentation/articles/introducing-your-site-navigation/",
```

```suggestion
      "to": "/documentation/articles/configure-your-site-navigation/",
```

Avoiding double redirect